src/DSharp.Mscorlib/Collections/IDictionary.cs

```diff
@@ -19,6 +19,7 @@ public interface IDictionary : IEnumerable
         [DSharpScriptMemberName("keyExists")]
         bool Contains(object key);
 
+        [DSharpScriptMemberName("addKeyValue")]
```

Comment:
Can you make sure this method actually exists in the ss script, I remember recently trying this and it wasn't working.

Reply:
done :)